src/data-table/types.js

```diff
@@ -106,14 +116,15 @@ export type StatefulContainerPropsT = {|
   ...StatefulDataTablePropsT,
   children: ({|
     filters: Map<string, {description: string}>,
-    onFilterAdd: (filterParams: {description: string}, title: string) => void,
+    onFilterAdd: (title: string, filterParams: {description: string}) => void,
```

Comment:
This would be a breaking change, right? Can we keep it as is?

Reply:
It's unstable and this particular prop is not used anywhere at uber so I think it's safe. The argument ordering was rather inconsistent so I took the opportunity to standardize it